packages/cli/src/lib/quire/index.js

```diff
@@ -129,20 +129,44 @@ async function initStarter (starter, projectPath) {
  */
 async function install(version='latest', options={}) {
   console.debug(`[CLI:quire] installing quire-11ty@${version}`)
-  fs.ensureDirSync(INSTALL_PATH)
+  const absoluteInstallPath = path.join(__dirname, 'versions')
```

Comment:
@anderspollack is this actually an _absolute_ path, would `path.isAbsolute(absoluteInstallPath)` be truthy? 
If this needs to be an absolute path we probably need something like `path.join(path.relative('/', __direname), 'version')`
which might be made into a utility module, perhaps 
```
import { isAbsolute, relative } from 'node:path'
export (path) => isAbsolute(path) ? path : relative('/', path)
```